coding/python_style_guide.rst

```diff
@@ -467,40 +474,42 @@ This comment should appear after the ``def`` line.
 =====================
 
 We follow `PEP 8ʼs naming conventions <https://www.python.org/dev/peps/pep-0008/#naming-conventions>`_, with exceptions listed here.
-The naming conventions for LSST Python and C++ source have been defined to be as similar as the respective languages allow.
+C++ source code included within a Python package would normally follow the naming conventions of the Python package for APIs that are to be visible to Python users.
 
-In general:
+All LSST Python source code is consistent with :pep:`8` naming in the following ways:
 
 - class names are ``CamelCase`` with leading uppercase,
 - module variables used as module global constants are ``UPPERCASE_WITH_UNDERSCORES``,
-- all other names are ``camelCase`` with leading lowercase.
+
+Some packages, for historical reasons, do not fully adhere to :pep:`8`.
+These packages, and the associated naming conventions, are described in :ref:`style-guide-py-sci-pi-naming`.
+Code in other packages SHOULD be consistent within a top-level package built by Jenkins, or within a distinct service, and it is RECOMMENDED that :pep:`8` naming convention be adopted for these, whilst understanding that it may be difficult to modify existing packages.
+Consistency within a package is important.
```

Comment:
Is this strong enough?  Consistency between packages in the same product is important, consistency within the same package would seem mandatory.